eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcingProtocol.scala

```diff
@@ -84,7 +84,7 @@ object EventsourcingProtocol {
   /**
    * Failure reply after a [[Replay]].
    */
-  case class ReplayFailure(cause: Throwable, instanceId: Int)
+  case class ReplayFailure(cause: Throwable, fromSequenceNr: Long, instanceId: Int)
```

Comment:
Please name this parameter like the corresponding one in `ReplaySuccess`. They are semantically equivalent.